src/machine.adoc

```diff
@@ -3233,47 +3233,49 @@ R/W/X permissions apply only to S and U modes.
 
 ===== Priority and Matching Logic
 
+On some implementations, misaligned loads, stores, and instruction
+fetches may be decomposed into multiple memory operations, some of which may
+succeed before an access-fault exception occurs (see <<memorymodel>>).
```

Comment:
<<memorymodel>> is probably a broken link since it's in the unpriv spec.  Maybe replace the parenthetical with ", as described in the RVWMO specification."

Reply:
Ooops I did notice this but then forgot (wrote this a while ago). I will update it. (*cough* #1395 :-D)

Reply:
Done

Reply:
#1395 is the long-term plan but we have bigger documentation fish to fry in the meantime.  It'll be a more involved effort than just concatenating the documents.